### PR TITLE
Fixed [Regional Board of the Plast/Governing Bodies/Governing body's …

### PR DESCRIPTION
**Pull request by christykobyletska**
…Sectors] Strict check in the description #1695, also #1120,#1621,added regex es to contacts

dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by OleksandrBilan — APPROVED** [review]

**Review by andrii52 — APPROVED** [review]

File: src/models/GllobalValidations/DescriptionValidation.tsx [hunk 1]
```diff
@@ -239,4 +239,14 @@ export const descriptionValidation = ({
...
+    DescriptionNotOnlyWhiteSpaces: [
+        {
+            pattern: /^\s*\S.*$/,
+            message: inputOnlyWhiteSpaces(),
+        },
+        { 
+            max: 1000, 
+            message: maxLength(1000),
+        },    
+    ],
```
Comment (Tali32) [inline src/models/GllobalValidations/DescriptionValidation.tsx:251]: Maybe it is better to change the previous method, because it looks almost the same?

Comment (christykobyletska) [reply src/models/GllobalValidations/DescriptionValidation.tsx:251]: can I delete description method?

Comment (christykobyletska) [reply src/models/GllobalValidations/DescriptionValidation.tsx:241]: Done

Comment (Tali32) [reply src/models/GllobalValidations/DescriptionValidation.tsx:241]: I was told about DescriptionAndQuestions Method) Description was using at some another place

Comment (christykobyletska) [reply src/models/GllobalValidations/DescriptionValidation.tsx:241]: misunderstanding) Description was used only there where I changed it to  DescriptionNotOnlyWhiteSpaces

Comment (christykobyletska) [reply src/models/GllobalValidations/DescriptionValidation.tsx:241]: DescriptionAndQuestions also says that field is mandatory, but we don't need this in description city/hovel/...

**Review by nicoolle — APPROVED** [review]

**Review by Tali32 — APPROVED** [review]

**Pull request merged**